localisation/braz_por/russia2_localisation_l_braz_por.yml

```diff
@@ -901,7 +901,7 @@
  SOV_core_baltic_state_desc:0 "[FROM.GetName] is part of our realm. We have to remind the people of the old days to earn their trust again."
  SOV_unhappy_peasants:0 "Unhappy Peasants"
  SOV_unhappy_peasants_desc:0 "The industrialization of the country required some sacrifices to be made, such as taxing the Peasants and taking their land away. The Kulaks, in particular, have been the target of the Five Year Plan since the start, aiming to reduce their number to none and force them to work in the cities. We have to keep an eye on them."
- SOV_peasants_rebelion:0 "The Peasants Strikes"
+ SOV_peasants_rebelion:0 "The Peasants' Strikes"
```

Comment:
SOV_peasants_rebelion:0 "The Peasants Strikes"
 SOV_peasants_rebelion_desc:0 "The Peasant's Strikes"